frontend/src/components/ui/copy-field.ts

```diff
@@ -0,0 +1,91 @@
+import { css, html } from "lit";
+import { customElement, property } from "lit/decorators.js";
+import { localized } from "@lit/localize";
+import { TailwindElement } from "@/classes/TailwindElement";
+import { classMap } from "lit/directives/class-map.js";
+
+/**
+ * Copy text to clipboard on click
+ *
+ * @example
+ * ```ts
+ * <btrix-copy-field label="my field" value=${value}></btrix-copy-field>
+ * ```
+ */
+@localized()
+@customElement("btrix-copy-field")
+export class CopyField extends TailwindElement {
```

Comment:
Do we still need to extend `TailwindElement` if we're just using `TailwindElement.styles`?

Reply:
That's a great question! Maybe not?

Reply:
Except that it turns out doing [this](https://github.com/webrecorder/browsertrix-cloud/pull/1427#issuecomment-1850670362) is a lot easier than I thought it would be — just a moment while I push